GERTi/GERTi Gateway.lua

```diff
@@ -8,15 +8,24 @@ local childNum = 1
 local connections = {}
 local connectDex = 1
 local tier = 0
-local handlers = {}
+local handler = {}
 
 if (not component.isAvailable("tunnel")) and (not component.isAvailable("modem")) then
     io.stderr:write("This program requires a network card to run.")
 end
-local modem = component.modem
-local tunnel = component.tunnel
-modem.setStrength(500)
-modem.open(4378)
+
+if (component.isAvailable("modem")) then
+    local modem = component.modem
```

Comment:
If a local is declared in this if...then block, will the modem variable be accessible outside? Same for the tunnel statement immediately below.